R/uti-CombineSpecs.R

```diff
@@ -0,0 +1,25 @@
+#' Combine Specifications
+#'
+#' This function combines a list of specifications into a single specification.
+#'
+#' @param lSpecs A list of specifications.
+#' @return A list representing the combined specification.
+#' @examples
+#' all_wf <- gsm::MakeWorkflowList()
```

Comment:
```suggestion
#' all_wf <- MakeWorkflowList()
```
Don't need the namespace within the package (and having it confuses RStudio if you try to F2 from function definition to function definition).